README.md

```diff
@@ -5,21 +5,27 @@
 
 The *dynatrace-sli-service* is a [Keptn](https://keptn.sh) service that is responsible for retrieving the values of Keptn-supported SLIs from a Dynatrace Metrics API endpoint.
 
-By default, the following SLIs are automatically supported:
+By default, even if you do not specifiy a custom SLI.yaml, the following SLIs are automatically supported:
 
  - Throughput
  - Error rate
  - Response time p50
  - Response time p90
  - Response time p95
 
+ These metrics are queried from a Dynatrace monitored Service entity with the tags keptn_project, keptn_service, keptn_stage & keptn_deployment.
+![](./images/defaultdynatracetags.png)
+
+ You can however define your own custom SLI.yaml where you are free in defining your own list of metrics coming from all monitored entities in Dynatrace (APPLICATION, SERVICE, PROCESS GROUP INSTANCE, HOST, CUSTOM DEVICE). More details on that further down in the readme.
+
 
 ## Compatibility Matrix
 
-| Keptn Version    | [Dynatrace-SLI-Service Service Image](https://hub.docker.com/r/keptncontrib/dynatrace-sli-service/tags) |
-|:----------------:|:----------------------------------------:|
-|       0.6.0      | keptncontrib/dynatrace-sli-service:0.3.0 |
-|       0.6.1      | keptncontrib/dynatrace-sli-service:0.4.0[^1] |
+| Keptn Version    | [Dynatrace-SLI-Service Service Image](https://hub.docker.com/r/keptncontrib/dynatrace-sli-service/tags) | Description |
```

Comment:
I'll remove the *Description*  column here. Describing or highlighting new features in the compatibility matrix will not scale. We should link to the release-notes in here.